_test/tests/inc/media_searchlist_test.php

```diff
@@ -0,0 +1,258 @@
+<?php
```

Comment:
Please rename this file to `media_searchlist.test.php`. Otherwise, these tests won't run.